webui/react/package.json

```diff
@@ -139,6 +139,7 @@
     "coveragePathIgnorePatterns": [
       "/src/services/api-ts-sdk/",
       ".stories.tsx"
-    ]
+    ],
+    "resetMocks": false
```

Comment:
**CRITICAL** this is an important jest config to prevent the mocks from being aggressively reset between tests.

Reply:
looks like there is an issue with the wait utils test that might be related to this.

Reply:
I'll wait for it to review

Reply:
it's been resolved (it might actually be an issue in its current form in the tip of the master, but the fix has applied here)